draft-ietf-httpbis-resumable-upload.md

```diff
@@ -681,6 +681,8 @@ The `104 (Upload Resumption Supported)` status code is can be used for two purpo
 - When responding to requests to create uploads, an interim response with the `104 (Upload Resumption Supported)` status code can be sent to indicate the server's support for resumable uploads, as well as the URI and limits of the corresponding upload resource in the `Location` and `Upload-Limit` header fields, respectively (see {{upload-creation}}). This notifies the client early about the ability to resume the upload in case of network interruptions.
 - While processing the content of a request to append representation data or create an upload, the server can regularly send interim responses with the `104 (Upload Resumption Supported)` status code to indicate the current upload progress in the `Upload-Offset` header field (see {{upload-creation}} and {{upload-appending}}). This allows the client to show more accurate progress information about the amount of data received by the server. In addition, clients can use this information to release representation data that was buffered, knowing that it doesn't have to be re-transmitted.
 
+When the client generates a 100-continue expectation because it wants an indication that the server is willing to accept the upload, it MAY treat an interim response with the `104 (Upload Resumption Supported)` status code as fulfilling the 100-continue expectation and start sending the request content. However, the server MUST NOT omit the `100 (Continue)` response because it has sent an interim response with the `104 (Upload Resumption Supported)` status code.
```

Comment:
To address #3210  would suggest making it clear what a client can do. E.g. State that it is allowed for a client to initiate a resumable upload (and an append?) with a 100-continue expectation. Then move on to saying what happens when it does as this para already says